concepts/comparisons/introduction.md

```diff
@@ -1,23 +1,65 @@
 # Introduction
 
-A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the values of two operands and returns `True` or `False` based on whether the `comparison` condition is met.
-The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, and `"!="`.
-They all have the same priority (which is higher than that of the Boolean operations)
+A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the _values_ of two [operands][operand] and returns a boolean `True` or `False` if the `comparison` condition is or is not met.
 
-## Comparison Chaining
+The table below shows the most common Python comparison operators:
 
-Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).
-This is also called `short-circuit` evaluation which means the execution is stopped if the truth value of the expression has already been determined.
-Note that the evaluation of expression takes place from left to right.
-In python, short circuiting is supported by various boolean operators, functions and, in this case, comparison chaining.
 
-## Comparison of different data types
+| Operator | Operation                | Description                                                    |
+| -------- | ------------------------ | -------------------------------------------------------------- |
+| `>`      | "greater than"             | `a > b` is `True` if `a` is **strictly** greater in value than `b` |
+| `<`      | "less than"                | `a < b` is `True` if `a` is **strictly** less in value than `b`    |
+| `==`     | "equal to"                 | `a == b` is `True` if `a` is **strictly** equal to `b` in value    |
+| `>=`     | "greater than or equal to" | `a >= b` is `True` if `a > b` OR `a == b` in value             |
+| `<=`     | "less than or equal to"    | `a <= b` is `True` if `a < b` or `a == b` in value             |
+| `!=`     | "not equal to"             | `a != b` is `True` if `a == b` is `False`                      |
+| `is`     | "identity"                 | `a is b` is `True` if **_and only if_** `a` and `b` are the same _object_  |
+| `is not` | "negated identity"                 | `a is not b` is `True` if `a` and `b` are **not** the same _object_ |

```

Comment:
Same here for alignment of column delimiter.

Reply:
Same comment as above.  The render is more important than the code -- and the render is in alignment.